msticpy/context/lookup.py

```diff
@@ -287,49 +321,49 @@ def browse_results(
 
         """
         if not isinstance(data, pd.DataFrame):
-            print("Input data is in an unexpected format.")
+            logger.info("Input data is in an unexpected format.")
             return None
         return browse_results(data=data, severities=severities, height=height)
 
     browse: Callable[..., SelectItem | None] = browse_results
 
-    def provider_usage(self) -> None:
+    def provider_usage(self: Self) -> None:
```

Comment:
same comment about stuff that is meant to be exposed to users being hidden in logger output.
Maybe we should have a generic output function that checks if it is in ipython and, if not logs the message, otherwise displays it.

Reply:
Change has been rolled back